the tdl binary doesn't exist, Debian already has one of them.

--- a/doc/devtodo.1.in
+++ b/doc/devtodo.1.in
@@ -21,7 +21,7 @@
 \fBtdd \fI<indices>\fP\fP
 Mark the specified items as being done.
 .TP 
-\fBtdl [\-g \fI<index>\fP] \fI<database>\fP\fP
+\fBtodo \-\-link [\-g \fI<index>\fP] \fI<database>\fP\fP
 Link the specified devtodo database into the current one, optionally grafting it as a child of the specified index.
 .SH "DESCRIPTION"
 .LP 
